.github/workflows/ci.yml

```diff
@@ -337,7 +342,7 @@ jobs:
           ./builder build -p ${{ env.PACKAGE_NAME }} --spec=downstream
       - name: Running samples in CI setup
         run: |
-          python3 -m pip install boto3
+          python3 -m pip install boto3 --break-system-packages
```

Comment:
Do we know why "--break-system-packages" is required for latest MacOS version? Would it be better to instead create and use a venv? Slightly worried that forcing the installation could cause a harder to track down problem in the future depending on what boto3 is overriding and/or the conflicts this is ignoring.